### PR TITLE
Feature/PRLC100-149

### PR DESCRIPTION
**Pull request by giorgi-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRLC100-149

### Change description ###
- Added /safety-concerns/orders-required/orders-required page
- 'Continue' button navigates the user to the same/current page 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/steps/urls.ts [hunk 1]
```diff
@@ -239,6 +239,8 @@ export const C100_C1A_SAFETY_CONCERNS_CONCERNS_ABOUT_CHILD: PageLink = `${C100_C
 export const C100_C1A_SAFETY_CONCERNS_REPORT_CHILD_ABUSE: PageLink = `${C100_C1A_SAFETY_CONCERNS_CHILD}/report-abuse`;
 export const C100_C1A_SAFETY_CONCERNS_CONCERNS_ABOUT_APPLICANT: PageLink = `${C100_C1A_SAFETY_CONCERNS_APPLICANT}/concerns-about`;
 export const C100_C1A_SAFETY_CONCERNS_REPORT_APPLICANT_ABUSE: PageLink = `${C100_C1A_SAFETY_CONCERNS_APPLICANT}/report-abuse`;
+export const C100_C1A_SAFETY_CONCERNS_ORDERS_REQUIRED: PageLink = `${C100_URL}/safety-concerns/orders-required`;
```
Comment (channa-patil) [inline src/main/steps/urls.ts:242]: @giorgi-hmcts Let's keep URL as 'c100-rebuild/safety-concerns/orders-required', it's repeated twice in the prototype.

**Review by channa-patil — CHANGES_REQUESTED** [review]
Please rebase with new c100 base branch & resolve merge conflicts.

File: src/main/steps/urls.ts (hunk 1 above)
Comment (giorgi-hmcts) [reply src/main/steps/urls.ts]: @channa-patil Done ✅

**Review by yogendraU — CHANGES_REQUESTED** [review]

**Comment by yogendraU** [comment]
@giorgi-hmcts , please fix the above review comments and resolve the conflicts as well.

File: src/main/steps/c100-rebuild/safety-concerns/orders-required/template.njk [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{% extends "common/template.njk" %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% block page_content %}
+<span class="govuk-caption-xl govuk-!-margin-top-7">{{caption}}</span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{title}}</h1>
```
[the remark this answers is not in the document]
Comment (giorgi-hmcts) [reply src/main/steps/c100-rebuild/safety-concerns/orders-required/template.njk]: Removed `govuk-!-margin-top-7` from the `h1` element ✅

File: src/main/steps/c100-rebuild/safety-concerns/orders-required/content.ts [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    c1A_keepingSafeStatement: {
+      type: 'textarea',
+      attributes: { rows: 10 },
+      validator: value => isFieldFilledIn(value) || isTextAreaValid(value),
```
[the remark this answers is not in the document]
Comment (giorgi-hmcts) [reply src/main/steps/c100-rebuild/safety-concerns/orders-required/content.ts]: Removed the extra `**isTextAreaValid**` validator ✅

**Review by channa-patil — APPROVED** [review]

**Pull request merged**